PalasoUIWindowsForms/Keyboarding/Windows/WinKeyboardAdaptor.cs

```diff
@@ -48,14 +49,124 @@ public LayoutName(string layout, string localizedLayout)
 			public string LocalizedName;
 		}
 
+		/// <summary>
+		/// This class receives notifications from TSF when the input method changes.
+		/// It also implements a fallback to Windows messages if TSF isn't available,
+		/// e.g. on Windows XP.
+		/// </summary>
+		private class TfLanguageProfileNotifySink : ITfLanguageProfileNotifySink
+		{
+			private readonly WinKeyboardAdaptor _keyboardAdaptor;
+			private List<Form> _toplevelForms = new List<Form>();
+
+			public TfLanguageProfileNotifySink(WinKeyboardAdaptor keyboardAdaptor)
+			{
+				_keyboardAdaptor = keyboardAdaptor;
+			}
+
+			#region ITfLanguageProfileNotifySink Members
+
+			public bool OnLanguageChange(ushort langid)
+			{
+				// In my tests we never hit this method (Windows 8.1). I don't know if the
+				// method signature is wrong or why that is.
+				return true;
```

Comment:
It might be helpful (though presumably there's doc of the interface somewhere) to comment what returning true indicates.